doc/changes.rst

```diff
@@ -1,6 +1,79 @@
 Changelog
 =========
 
+Release v0.2.1 (2020/XX/XX)
+---------------------------
+
+.. image:: https://zenodo.org/badge/DOI/10.5281/zenodo.4025418.svg
+   :alt: Digital Object Identifier for the Zenodo archive
+   :target: https://doi.org/10.5281/zenodo.4025418
+
+Highlights
+
+* 🎉 **Patch release with more tutorials and gallery examples!** 🎉
+* 📹 Add `Liam <https://github.com/liamtoney>`__'s `ROSES 2020 PyGMT talk <https://www.youtube.com/watch?v=SSIGJEe0BIk>`__ (#643)
```

Comment:
This depends on whether we make a v0.2.2 release, but should we add a statement about Python and GMT version support going forward (#644)? I.e. something like:

- PyGMT v0.2.x will be the last version to support Python 3.6, future versions of PyGMT(v0.3.x and above) will only support Python 3.7-3.9+
- PyGMT v0.2.x will be the last version to support GMT 6.1.1, future versions of PyGMT (v0.3.x and above) will require GMT 6.2.0 as a minimum.

Or we can do it in v0.2.2 (scheduled for December)?

Reply:
> we can do it in v0.2.2 (scheduled for December)?

GMT 6.2 is most likely to be released in January 2021 (https://forum.generic-mapping-tools.org/t/gmt-community-zoom-meeting-thursday-oct-22-utc-1700-hst-0700/943/6). There are at least two months before that. v0.2.2 in December (before AGU?) sounds good for a patch release. We may even have v0.2.3 before the release of GMT 6.2, depending on the changes we have made.

Reply:
Ok, let's not add this notice then until PyGMT v0.2.2. I think one more patch release for the PyGMT v0.2.x series before GMT 6.2.0 comes out is enough (unless another maintainer steps up to do the work). Personally I *really* can't spend too much time maintaining PyGMT for the next two months until my thesis is finished :sweat_smile:

Reply:
Thesis always has the highest priority. 😃